civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6191,6 +6191,34 @@ function _get_task_filter_by_date($date, $normalize = false) {
   return $normalize ? ($filter == 'tomorrow' ? 'week' : $filter) : $filter;
 }
 
+/**
+ * Sets the maximum viewport scale to 1 to prevent auto zooming on input focus
+ * @see https://stackoverflow.com/questions/11064237/prevent-iphone-from-zooming-form
+ *
+ * @param object $head_elements
+ */
+function _set_maximum_scale_to_viewport_meta_tag(&$head_elements) {
+  foreach ($head_elements as $key => $tag) {
+    if (isset($tag['#attributes']['name']) && $tag['#attributes']['name'] === 'viewport') {
+      $head_elements[$key]['#attributes']['content'] .= ', maximum-scale=1';
```

Comment:
Its better to check if maximum-scale is already present. I know its not now, but incase we add it from somewhere else or we update the theme later, there is a chance of having this twice.

Reply:
Done in the new PR